### PR TITLE
fix: change order of merging configs

### PR DESCRIPTION
**Pull request by djamaile**
baseUrl doesn't get replaced:
```sh
❯ yarn backstage-cli config:print --config app-config.deployment.yaml --config app-config.yaml --frontend
yarn run v1.22.19
$ /Users/djamailer/Desktop/projects/backstage-changeset/node_modules/.bin/backstage-cli config:print --config app-config.deployment.yaml --config app-config.yaml --frontend
Loaded config from app-config.deployment.yaml, app-config.yaml
app:
  baseUrl: http://localhost:3000
  title: Scaffolded Backstage App
backend:
  baseUrl: http://localhost:7007
organization:
  name: My Company
integrations:
  github:
    - host: github.com
techdocs:
  builder: local
catalog:
  import:
    entityFilename: catalog-info.yaml
    pullRequestBranchName: backstage-integration

✨  Done in 1.02s.
```

baseUrl gets properly replaced:
```sh
❯ yarn backstage-cli config:print --config app-config.yaml --config app-config.deployment.yaml --frontend
yarn run v1.22.19
$ /Users/djamailer/Desktop/projects/backstage-changeset/node_modules/.bin/backstage-cli config:print --config app-config.yaml --config app-config.deployment.yaml --frontend
Loaded config from app-config.yaml, app-config.deployment.yaml
app:
  title: Scaffolded Backstage App
  baseUrl: http://test
organization:
  name: My Company
backend:
  baseUrl: http://test
integrations:
  github:
    - host: github.com
techdocs:
  builder: local
catalog:
  import:
    entityFilename: catalog-info.yaml
    pullRequestBranchName: backstage-integration

✨  Done in 1.22s.
```

### CODE REVIEW

File: src/programs/aws.ts [hunk 1]
```diff
@@ -186,7 +186,7 @@ export const AWSProgram = (opts: OptionValues) => {
           };
 
     const CONTAINER_SERVICE_URL = containerService.url.apply(url =>
-      url.endsWith('/') ? url.slice(0, -1) : url
+      url.endsWith('/') ? url.slice(0, -1) : url,
```
Comment (mitchhentgesspotify) [inline src/programs/aws.ts]: While you're here, do you mind making `CONTAINER_SERVICE_URL` -> `containerServiceUrl`? I didn't want to bother a contributor with this nit, but I'm happy to bug you with it 😉

**Review by mitchhentgesspotify — APPROVED** [review]
This seems purely style and like a no-op, I'm surprised that it's changing behaviour - isn't the callback the first parameter either way? Weird.

File: src/programs/aws.ts (hunk 1 above)
Comment (djamaile) [reply src/programs/aws.ts]: Done in https://github.com/backstage/backstage-deploy/pull/33/commits/094203933370acec9c9a5281c5b7dc62ee635c60

File: templates/pulumi/Dockerfile [hunk 2]
```diff
@@ -37,4 +37,4 @@ RUN echo "techdocs:" >> app-config.deployment.yaml
 RUN echo "    generator:" >> app-config.deployment.yaml
 RUN echo "        runIn: 'local'" >> app-config.deployment.yaml
 
-CMD ["node", "packages/backend", "--config",  "app-config.deployment.yaml", "--config", "app-config.yaml"]
```
Comment (mitchhentgesspotify) [inline templates/pulumi/Dockerfile:40]: What in tarnation is the change here? Weird why this is showing up in the diff.

**Review by mitchhentgesspotify — APPROVED** [review]

**Pull request merged**